papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_3.yaml

```diff
@@ -41,7 +41,7 @@ spec:
             python setup.py develop;
             cd ../zdm;
             git fetch;
-            git checkout H0_paper;
```

Comment:
I think even my 2nd year students could have guessed what I am going to say here! ;-)